### PR TITLE
FIX import statement for SMAC>0.13

### PR DESCRIPTION
**Pull request by KEggensperger**
Should be merged together with https://github.com/automl/ParameterImportance/pull/130

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -13,3 +13,4 @@ scipy
 sklearn
 selenium
 nose
+smac>=0.13.0
```
Comment (shukon) [inline requirements.txt:16]: smac is already mentioned in line 3

**Review by shukon — CHANGES_REQUESTED** [review]
Looks good except for requirements, though I didn't run it yet. Will try to in the next week. Could also try to not break backwards-compatibility, but not sure if that's necessary at all (don't know the context CAVE is used atm)

File: requirements.txt (hunk 1 above)
Comment (KEggensperger) [reply requirements.txt]: Sorry, I fixed this.

**Comment by shukon** [comment]
@KEggensperger do you want me to merge this here and in automl/ParameterImportance#130 ? Or is someone responsible for these packages?

**Review by AndreBiedenkapp — APPROVED** [review]
Thanks for taking the time to making it work :)

**Pull request merged**